roles/idrac_os_deployment/templates/UBUNTU_jammy.j2

```diff
@@ -0,0 +1,43 @@
+#cloud-config
+# vim: ft=yaml:
+autoinstall:
+  version: 1
+  early-commands:
+    - ["cat", "/autoinstall.yaml"]
+  refresh-installer:
+    update: yes
+  apt:
+    preserve_sources_list: false
+    mirror-selection:
```

Comment:
do we have to set this at all? maybe just remove this section and see if it works... ?